.eslintrc.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+    jest: true
+  },
+  extends: [
+    '@nimblehq/eslint-config-nimble',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:jsx-a11y/recommended',
+    'plugin:import/errors',
+    'plugin:prettier/recommended'
+  ],
+  overrides: [
+    {
+      files: 'src/tests/**/*.test.ts',
```

Comment:
Where will you put the test files? Will they be in `tests` folder or in the same directory as its code?